components/OfferSearch.tsx

```diff
@@ -18,12 +19,16 @@ export type OfferSearchProps = {
 	offerType: Record<string, any>;
 };
 
+const CZECH = "cs";
+
```

Comment:
maybe we should extract these language constants to separate `constants.ts` file

Reply:
yes, you are right.. added `utils` folder and `constants.ts` file